### ISSUE TITLE
fix: give full with to PhoneInputField  <Input />

### ISSUE THREAD

**Issue by jonoise** [lifespikes/packages#25]
Using the `PhoneInputField` the <Input /> is not taking full width

![Screenshot 2024-05-13 at 19 14 29](https://github.com/lifespikes/packages/assets/71573508/76151320-c2bf-4b67-883f-c43651a1713e)

After digging the code i found you can pass this `containerComponent` and `inputComponent` props. The problems seems to be with the `inputComponent`

<img width="456" alt="Screenshot 2024-05-13 at 19 17 16" src="https://github.com/lifespikes/packages/assets/71573508/5c122064-e460-4d6e-9d6d-2167036c6cd9">

After setting a custom one and styling it properly it takes full width! 

![Screenshot 2024-05-13 at 19 18 29](https://github.com/lifespikes/packages/assets/71573508/903f4058-bbf1-4dbe-840e-53697fe193a2)

But this will not allow us to actually change the value, it works only visually.

i think the simplest approach to is give the input/input-container (if theres any input-container) a full width by default or to expose `<Input/>` props

**Comment by santigp258** [comment]
@jonoise hey, bro. Thanks for reporting this. You can do the following, which will be valid for the new version of lifespikes ui:

The solution was to add a property for the Input component called containerClassName

```tsx

import {
  Form,
  PhoneInputField,
  Input,
  InputProps
} from '@lifespikes/ui';

const InputComponent = React.forwardRef<HTMLInputElement, InputProps>(
  ({ className, ...props }, ref) => (
    <Input
      className={cn('rounded-e-lg rounded-s-none flex-1', className)}
      containerClassName="w-full" // new prop added
      {...props}
      ref={ref}
    />
  )
);
InputComponent.displayName = 'InputComponent';


const App = ()=>{
     return (
       <Form>
      <div className="w-[400px] bg-red-500">
        <PhoneInputField
          name="phone_input_field"
          label="Phone"
          _phoneInput={{
            inputComponent: InputComponent,
          }}
        />
      </div>
    </ExampleForm>
     )
}
```

**Comment by santigp258** [comment]
@jonoise fixed in version [1.7.1](https://www.npmjs.com/package/@lifespikes/ui)
